README.rst

```diff
@@ -1,11 +1,11 @@
 cloud-pubsub-logging-python
 ===========================
 
-    A logging handler for publishing the logs to Cloud Pub/Sub.
+    Logging handlers for publishing the logs to Cloud Pub/Sub.
 
 |pypi| |build| |coverage|
 
-You can use the pubsub_logging.PubsubHandler to publish the logs to `Cloud Pub/Sub`_. You can use this module with `the standard Python logging module`_.
+You can use the pubsub_logging.PubsubHandler or pubsub_logging.AsyncPubsubHandler to publish the logs to `Cloud Pub/Sub`_. You can use this module with `the standard Python logging module`_.
```

Comment:
backtics around `pubsub_logging.PubsubHandler` and `pubsub_logging.AsyncPubsubHandler`?